src/service/resources-impl.js

```diff
@@ -225,8 +225,9 @@ export class ResourcesImpl {
     divertIntersectResources(this.win);
 
     if (
-      getExperimentBranch(this.win, INTERSECT_RESOURCES_EXP.id) ===
-        INTERSECT_RESOURCES_EXP.experiment &&
+      (isExperimentOn(this.win, 'bento') ||
```

Comment:
Would we technically want the intersect-resources exp to be on if any `bento-*` exp is on? particularly carousel and lightbox right?

Reply:
I think I'd like it only for the global experiment. It's a more direct indication that bento mode is desired. But I think we'll be adjusting as we learn more.